### DISCUSSION TITLE
Loading component not showing in Chromium browser

### DISCUSSION THREAD

**Discussion by thevalo** [saadeghi/daisyui#2358] [Q&A] [answered]
Hi, I'm using latest daisyUI version as for this date. (v3.7.5).

I used a loading component for example as `<span class="loading loading-spinner loading-lg"></span>`, though the spinner is not showing. When you inspect element:
![image](https://github.com/saadeghi/daisyui/assets/28608203/b7ecb212-372d-4331-b4c4-7b3c98825630)

As far as I can comprehend, it needs the `-webkit-` prepended, but this does not happen automatically. On the [documentation](https://daisyui.com/components/loading/) I can see the `-webkit-`. 

What is happening on my end?

**Comment by saadeghi** [comment, marked as answer]
I think you didn't setup `autoprefixer`
```
npm i -D autoprefixer
```
```js
// postcss.config.js
module.exports = {
  plugins: {
    tailwindcss: {},
    autoprefixer: {},
  }
}
```

**Reply by thevalo** [reply]
Seems like reinstalling `autoprefixer` did the job. Gracias!
